### PR TITLE
building PELUX: add table with variants/images

### PR DESCRIPTION
**Pull request by tobsan**
Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 1]
```diff
@@ -41,6 +41,32 @@ Then make repo tool fetch all sources using the command `repo sync`.
...
++=====================================+============+==============+============+============+
+| core-image-pelux-minimal            | :green:`✔` | :green:`✔`   | :green:`✔` | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+| core-image-pelux-minimal-dev        | :green:`✔` | :green:`✔`   | :green:`✔` | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+| core-image-pelux-qtauto-neptune     | :red:`✘`   | :green:`✔`   | :red:`✘`   | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+| core-image-pelux-qtauto-neptune-dev | :red:`✘`   | :green:`✔`   | :red:`✘`   | :green:`✔` |
++-------------------------------------+------------+--------------+------------+------------+
+
```
Comment (GordanM) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: Nice table! I am wondering if we should have the experimental table below or is that something we don't want to point out?

**Review by jonte — APPROVED** [review]

**Review by GordanM — APPROVED** [review]

File: docs/chapters/baseplatform/building-PELUX-sources.rst (hunk 1 above)
Comment (rpannek) [reply docs/chapters/baseplatform/building-PELUX-sources.rst]: I'm not a fan of css-classes which tell how things look like, instead they should be semantically named and say what something is like "available" and "not-available", if we then change the colors some day in the future we only need to change the color, not the class names all over the table.

**Review by rpannek — CHANGES_REQUESTED** [review]

File: docs/chapters/baseplatform/building-PELUX-sources.rst (hunk 1 above)
Comment (tobsan) [reply docs/chapters/baseplatform/building-PELUX-sources.rst]: Good point, I will indeed change the names to something more semantic.

Comment (tobsan) [reply docs/chapters/baseplatform/building-PELUX-sources.rst]: @rpannek I updated it now with better names!

**Review by rpannek — APPROVED** [review]

**Comment by tobsan** [comment]
@GordanM I spoke to @jonte and I think the two of us at least agreed that having the experimental stuff in such a table takes them from seemingly experimental to more or less supported, in the eye of a reader.

**Comment by GordanM** [comment]
@tobsan fine with me then.

**Comment by rpannek** [comment]
@tobsan not if you add a ⚗ to your ✘✔ ;-)

**Pull request merged**